src/styles/components/_trending.scss

```diff
@@ -0,0 +1,42 @@
+.trending {
+  max-width: 30%;
+  flex: 1 1 auto;
+
+  .article-list{
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    overflow: scroll;
```

Comment:
But do we really want overflow here at all? I feel like people don't want to scroll a list. Why not just let it finish expanding in the y direction?